src/test/java/ca/corefacility/bioinformatics/irida/ria/integration/projects/ProjectMembersPageIT.java

```diff
@@ -64,4 +69,58 @@ public void testCollaborator() {
 		ProjectMembersPage page = ProjectMembersPage.goTo(driver());
 		assertFalse("Add Members button should not be visible", page.isAddMemberBtnVisible());
 	}
+
+	@Test
+	public void testRemoteProjectManagerPageSetup() {
```

Comment:
Thanks a bunch for adding the tests for this.

Reply:
No problem. Thought it would be a good idea to have them :)